src/DvsaLogger/Service/DoctrineQueryLoggerService.php

```diff
@@ -2,49 +2,69 @@
 
 namespace DvsaLogger\Service;
 
-use Doctrine\DBAL\Logging\SQLLogger;
-use Doctrine\ORM\EntityRepository;
 use DvsaLogger\Debugger\BacktraceDebugger;
 use Laminas\Log\LoggerAwareInterface;
 use Laminas\Log\LoggerAwareTrait;
+use Doctrine\DBAL\Types\Type;
 
 /**
  * Class DoctrineLoggerService
  *
  * @package DvsaDoctrineLogger\Service
  */
-class DoctrineQueryLoggerService implements SQLLogger, LoggerAwareInterface
+class DoctrineQueryLoggerService implements LoggerAwareInterface
 {
     use LoggerAwareTrait;
 
+    /** @var bool */
     protected $enabled;
 
+    /** @var string */
     protected $sql;
+    /** @var array|null */
     protected $params;
+    /** @var array|null */
     protected $types;
+    /** @var float */
     protected $startTime;
+    /** @var BacktraceDebugger */
     protected $debugger;
 
+    /**
+     * @param \Laminas\Log\LoggerInterface $logger
+     * @param bool $enabled
+     */
     public function __construct($logger, $enabled = false, BacktraceDebugger $debugger = null)
     {
+        $this->sql = '';
+        $this->startTime = 0;
```

Comment:
These two lines are somewhat arbitrary to stop a "not set in constructor" error. Are people fine with this, or would they rather the error was suppressed?